src/main/java/seedu/address/logic/commands/event/DeleteEventCommand.java

```diff
@@ -14,7 +14,7 @@
 import seedu.address.ui.MainWindow;
 
 /**
- * Deletes a event identified using it's displayed index from the address book.
+ * Deletes a event identified using it's displayed index from the EventBook.
```

Comment:
Deletes an event identified using its* displayed index from the EventBook.

Reply:
Edited, thanks!